pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -2100,3 +2123,74 @@ func getRouteHostname(obj interface{}) (host string) {
 	}
 	return host
 }
+
+func checkAndCreateAdminSecret(service kubernetes.PlatformService, namespace string, secretName string, environment api.EnvironmentType) (error, string, string) {
+	/* The default secret will be like this
+
+	apiVersion: v1
+	kind: Secret
+	metadata:
+	  name: kie-admin-credentials
+	type: Opaque
+	data:
+	  //adminUser
+	  username: YWRtaW4=
+	  //RedHat
+	  password: UmVkSGF0
+
+	*/
+
+	customerSecret, err := getSecret(service, namespace, secretName)
```

Comment:
better to rename it, something like providedSecret

Reply:
ok